qase-playwright/src/reporter.ts

```diff
@@ -302,4 +318,23 @@ export class PlaywrightQaseReporter implements Reporter {
   public static addIds(ids: number[], title: string): void {
     this.qaseIds.set(title, ids);
   }
+
+  /**
+   * @param {(string | Buffer)[]} logs
+   * @param {string} name
+   * @returns {Attachment}
+   * @private
+   */
+  private convertLogsToAttachments(logs: (string | Buffer)[], name: string): Attachment {
+    let content = '';
+    for (const line of logs) {
+      content = content + line.toString();
+    }
```

Comment:
You can join them all at once, without one-by-one concatenation.
```js
logs.map(line => line.toString()).join('\n')

Reply:
fixed